port/test/u_port_test.c

```diff
@@ -1511,7 +1514,7 @@ U_PORT_TEST_FUNCTION("[port]", "portOs")
     U_PORT_TEST_ASSERT(errorCode == 0);
     U_PORT_TEST_ASSERT(gTaskHandle != NULL);
 
-    U_TEST_PRINT_LINE("time now %d ms.", (int32_t) uPortGetTickTimeMs());
+    U_TEST_PRINT_LINE("time now %d ms.", uPortGetTickTimeMs());
```

Comment:
should this all have relative times to the start ? use uTimeoutStart and elapsed ? 
Just consistency not anything wrong here.

Reply:
I found it useful to have the timestamps in front of me when initially debugging this, kind of easier to visualise.

Reply:
I see, you want to see how the packet timing when the go out, and at the same time wouldn't an elapsed ms time since start be even more useful, so it keeps the numbers less random and small ?